specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/2021-03-01/connectedClusters.json

```diff
@@ -0,0 +1,698 @@
+            },
+            "resource": {
+              "type": "string",
+              "description": "Connected Cluster Resource on which the operation is performed"
+            },
+            "operation": {
+              "type": "string",
+              "description": "Operation type: Read, write, delete, etc."
+            },
+            "description": {
+              "type": "string",
+              "description": "Description of the operation."
+            }
+          }
+        }
+      }
+    },
+    "ConnectedClusterList": {
+      "description": "The paginated list of connected Clusters",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of connected clusters",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConnectedCluster"
+          }
+        },
+        "nextLink": {
+          "description": "The link to fetch the next page of connected cluster",
+          "type": "string"
+        }
+      }
+    },
+    "ConnectedCluster": {
+      "type": "object",
+      "required": [
+        "properties",
+        "identity"
+      ],
+      "properties": {
+        "identity": {
+          "$ref": "#/definitions/ConnectedClusterIdentity",
+          "description": "The identity of the connected cluster."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ConnectedClusterProperties",
+          "description": "Describes the connected cluster resource properties."
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData",
+          "readOnly": true,
+          "description": "Metadata pertaining to creation and last modification of the resource"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../common-types/resource-management/v2/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Represents a connected cluster."
+    },
+    "ConnectedClusterIdentity": {
+      "type": "object",
+      "required": [
+        "type"
+      ],
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The principal id of connected cluster identity. This property will only be provided for a system assigned identity."
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant id associated with the connected cluster. This property will only be provided for a system assigned identity."
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of identity used for the connected cluster. The type 'SystemAssigned, includes a system created identity. The type 'None' means no identity is assigned to the connected cluster.",
+          "enum": [
+            "None",
+            "SystemAssigned"
```

Comment:
I believe to support background compatibility, we cannot remove none here. This is something we should discuss with RPaaS team.